src/main/resources/whereabouts/patterns/BCI.properties

```diff
@@ -0,0 +1,26 @@
+# Buckley Hall
+BCI_A-Wing=BCI-A-.+
+BCI_A-Wing_Landing\ 1+2=BCI-A-[1-2]-.+
+BCI_A-Wing_Landing\ 3+4=BCI-A-[3-4]-.+
+BCI_A-Wing_Landing\ 1=BCI-A-1-.+
+BCI_A-Wing_Landing\ 2=BCI-A-2-.+
+BCI_A-Wing_Landing\ 3=BCI-A-3-.+
+BCI_A-Wing_Landing\ 4=BCI-A-4-.+
+BCI_B-Wing=BCI-B-.+
+BCI_B-Wing_Landing\ 1+2=BCI-B-[1-2]-.+
```

Comment:
@Mjwillis @GurnankCheema 
I had a quick HUDDLE with @GurnankCheema  in order to understand the properties why are so long and in my opinion repetitive.
as per my understanding is cause NOMIS is using those patterns and it is a work around it.